demisto_sdk/commands/common/hook_validations/incident_field.py

```diff
@@ -220,6 +220,11 @@ def is_valid_name(self):
             "XDR Low Severity Alert Count",
             "XDR Incident ID",
             "Detection Ticketed",
+            "Claroty Alert Resolved",  # Needed for incidentfield-Claroty_Alert_Resolved.json
```

Comment:
add todo here to remove these when https://github.com/demisto/etc/issues/24232 is revolsed